dll/win32/kernel32/kernel32.spec

```diff
@@ -32,7 +32,7 @@
 @ stub -version=0x600+ BaseGenerateAppCompatData
 @ stdcall BaseInitAppcompatCacheSupport()
 @ stdcall BaseIsAppcompatInfrastructureDisabled() IsShimInfrastructureDisabled
-@ stdcall -version=0x501-0x502 BaseProcessInitPostImport()
+@ stdcall BaseProcessInitPostImport()
```

Comment:
Unwanted change?

Reply:
No; see my commit comment for 84de3f2235675b5d73850dca0be61d91b3344b2c for why this is needed.

Reply:
That looks like a wrong fix as well:
either this function/export actually exists on NT6+ and this change should be a separate PR,
or it does not and there should be no change.

Reply:
does not really matter, but reactos components should be able to bypass this mechanism so either way should work,
removing this version or leaving it in place.

Reply:
They should all be 0x600+, so that the loader can identify DLLs that have only Vista+ exports and fail when trying to load the DLL, emulating the non-existance of the DLL for apps that check this to see, whether it's running on Vista.

Reply:
> They should all be 0x600+

Which 'they' are you referring to? Did you mean to comment about the apisets instead?
As changing `BaseProcessInitPostImport` to `0x600+` from `0x501-0x502` seems even more unexpected...